packages/@pollyjs/core/tests/browser/unit/utils/serialize-request-body-test.js

```diff
@@ -23,24 +23,31 @@ describe('Unit | Utils | serializeRequestBody', function() {
   });
 
   it('should handle files', async function() {
-    expect(await serializeRequestBody(new File(['file'], 'file.txt'))).to.equal(
-      `data:;base64,${btoa('file')}`
-    );
+    expect(
+      await serializeRequestBody(
+        new File(['file'], 'file.txt', {
```

Comment:
> readAsDataURL function is supported, but the mediatype in Chrome depends on entry name extension, mediatype in IE is always empty (which is the same as text-plain according the specification), the mediatype in Firefox is always application/octet-stream. For example, if the content is abcdefg then Firefox returns data:application/octet-stream;base64,YWJjZGVmZw==, IE returns data:;base64,YWJjZGVmZw==, Chrome returns data:<mediatype depending on extension of entry name>;base64,YWJjZGVmZw==.

Seems how the type is negotiated when not supplied is a browser quirk.  In Chrome 68 if the type of `''` it would not default to `application/octet-stream` - now it does... like Firefox.